src/ipsis/course-merge/CourseMergeOfferingEntity.js

```diff
@@ -67,6 +67,10 @@ export class CourseMergeOfferingEntity extends Entity {
 		return this._entity?.hasActionByName(Actions.ipsis.sisCourseMerge.selectAsTarget);
 	}
 
+	hasContent() {
+		return this._entity?.properties?.hasContent;
+	}
```

Comment:
Nit: could we move this up to where the other properties are?

Reply:
Good idea, done.

Reply:
What do you mean by `"but would like to see the change."`